orc8r/lib/go/initflag/syslog.go

```diff
@@ -1,161 +0,0 @@
-/*
-Copyright 2020 The Magma Authors.
-
-This source code is licensed under the BSD-style license found in the
-LICENSE file in the root directory of this source tree.
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package initflag
-
-import (
-	"bufio"
-	"flag"
-	"fmt"
-	"log"
-	"log/syslog"
-	"os"
-	"path/filepath"
-	"strings"
-)
-
-const syslogFlag = "stderr2syslog"
```

Comment:
This seems to be used in the magma openwrt deployment here:
./openwrt/gateway/configs/etc/init.d/magma.template

Reply:
I do not know anything about this openwrt stuff.